user/mods/Trap-PathToTarkov/config/config.json

```
@@ -300,16 +307,94 @@
         "ru": "Somewhere in the Military reserve"
       },
       "access_via": ["ReserveBunkerStash"]
+    },
+    "Svetlana": {
+      "// mod integration for Svetlana": true,
+      "// Svetlana is available at MechanicStash": true,
+      "disable_warning": true,
```

```suggestion
      "disable_warning": false,
```

I think this should be false, it's possible than the load order has an impact and with the warning we can be aware of an issue.

I put this to true in a default config when some people may havn't downloaded the custom trader (and so this avoid polluting his logs with unnecessary warning)